src/Compilers/CSharp/Test/Semantic/Semantics/BindingAsyncTasklikeMoreTests.cs

```diff
@@ -162,6 +162,245 @@ .locals init (C.<G>d__1<T> V_0)
 }");
         }
 
+        [Fact]
+        public void AsyncMethod_NullBuilder()
```

Comment:
>AsyncMethod_NullBuilder [](http://example.com/codeflow?start=20&length=23)

It looks like we're already testing this case in `AsyncTasklikeBadAttributeArgument4()`.